src/status_im/contexts/wallet/effects.cljs

```diff
@@ -1,7 +1,19 @@
 (ns status-im.contexts.wallet.effects
-  (:require [re-frame.core :as rf]
-            [react-native.share :as share]))
+  (:require
+    [clojure.string :as string]
+    [native-module.core :as native-module]
+    [re-frame.core :as rf]
+    [react-native.share :as share]))
 
 (rf/reg-fx :effects.share/open
```

Comment:
unrelated to this pr but I think we should move this effect, it's clearly more generic than wallet 👍

Reply:
Hmm.. maybe.. but I am not aware for somewhere else in the app where we generate keypairs besides wallet

Reply:
I'm referring to `:effects.share/open` 
generate key pairs is fine 👍